test/functional/normal/fold_spec.lua

```diff
@@ -44,4 +47,114 @@ describe('Folds', function()
     8
     9]])
   end)
+  it('updates correctly on :read', function()
+    -- luacheck: ignore
```

Comment:
Since this is becoming more common, let's make it granular (I think this wasn't supported in older version of luacheck, but it is now in the version we use). warning 621 is "inconsistent indentation", we can ignore it like this:

    -- luacheck: ignore 621

Full list of warnings: http://luacheck.readthedocs.io/en/stable/warnings.html